src/mg_ore.cpp

```diff
@@ -64,8 +64,8 @@ size_t OreManager::placeAllOres(Mapgen *mg, u32 blockseed,
 
 void OreManager::clear()
 {
-	for (size_t i = 0; i < m_objects.size(); i++) {
-		Ore *ore = (Ore *)m_objects[i];
+	for (auto &object : m_objects) {
```

Comment:
auto should be used essential for iterators to reduce litterature of the iterator name, but here i should replace it